### ISSUE TITLE
Test Results Not Matching Output Display

### ISSUE THREAD

**Issue by lmack82** [FreeCodeCamp/FreeCodeCamp#7848]
#### Issue Description
Issue happens upon running test. Check one says:

> find([1, 3, 5, 8, 9, 10], function(num) { return num % 2 === 0; }) should return 8.

The preview window after running the test shows 8, yet this is not being counted. I know my code so far isn't the best way to approach this, but I thought it was odd that I returned what it asked for and it didn't acknowledge that. I was wondering if there might be a bug here.

#### Browser Information
Mozilla Firefox 45.0.1
Windows 7

#### Your Code

```js
function find(arr, func) {
  var num = arr.filter(func);
  num = num.slice(0,1);
  num = num.join('');
  return num;
}

find([1, 3, 5, 8, 9, 10], function(num) { return num % 2 === 0; });
```

#### Screenshot
![temp](https://cloud.githubusercontent.com/assets/17714490/14216840/c7bf5954-f7ff-11e5-9e56-3464b908f345.png)

**Comment by erictleung** [comment]
Please take all coding questions to the [Help Room](https://gitter.im/FreeCodeCamp/HelpJavaScript). A hint to your problem is checking the data type of your function output. Happy coding!

**Comment by lmack82** [comment]
This isn't a coding question. This is a discrepancy between the display and results. I am not seeking help with coding at the moment. I'm still trying to solve the problem myself. I'm only pointing out that the results screen shows that 8 is returned but the test that says it should return 8 says otherwise.

To be clear: I am _not_ asking for help on the solution nor am I asking why my code fails to past the test. I am asking how can I test my code if the display is inconsistent with the test?

In other words, if it says "find([1, 3, 5, 8, 9, 10], function(num) { return num % 2 === 0; }) should return 8." and the test says it does not, I need to know what is actually being returned in order to figure out how to fix my code. The only way I know of doing this is to paste in "find([1, 3, 5, 8, 9, 10], function(num) { return num % 2 === 0; })" and see what gets returned. When I do this, it says 8, which is in conflict with what I was just told.

This is why I reported this as a possible bug.

**Comment by erictleung** [comment]
@lmack82 You've returned your answer as a string but it should really be a number.
